src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java

```diff
@@ -718,4 +726,61 @@ public void testFindAllStructureIdsShouldLimitByGivenParam() {
 		assertEquals(1542376382851l, idsModel.getRight().longValue());
 	}
 	
+	@Test
+	public void testSearchlLocations() {
+		LocationTagMap locationTagMap = new LocationTagMap();
+		locationTagMap.setLocationId(2l);
+		locationTagMap.setLocationTagId(2l);
+		LocationSearchBean locationSearchBean = new LocationSearchBean();
+		locationSearchBean.setName("a");
+		locationSearchBean.setLocationTagId(2l);
+		locationSearchBean.setPageSize(20);
+		locationSearchBean.setPageNumber(1);
+		locationSearchBean.setOrderByFieldName("id");
+		locationSearchBean.setOrderByType(OrderByType.ASC);
+		locationTagRepository.addLocationTagMap(locationTagMap);
+		List<CustomPhysicalLocation> locations = new ArrayList<CustomPhysicalLocation>();
+		locations = locationRepository.searchLocations(locationSearchBean);
+		assertNotNull(locations);
+		assertEquals(1l, locations.size());
```

Comment:
add a test that e.g tests that no data is returned e.g if name or location tag does not exist